tests/ProxyManagerTest/Factory/LazyLoadingGhostFactoryTest.php

```diff
@@ -102,6 +102,33 @@ public function testWithOptionalFactory()
     }
 
     /**
+     * @dataProvider skipPropertiesFixture
+     */
+    public function testSkipProperties($className, $properties)
+    {
+        $propertyName = $properties[0]; 
+        $factory      = new LazyLoadingGhostFactory();
+        $ghostObject  = $factory->createProxy(
+            $className,
+            function () {
+                throw new \RunTimeException('Teste Failing every time');
+            },
+            $properties
+        );
+
+        $ghostObject->$propertyName;
```

Comment:
This access will fail for private/protected properties. Also, you should assert on the value of the property once retrieved